server/cluster.go

```diff
@@ -110,7 +109,7 @@ func (c *RaftCluster) start() error {
 		return nil
 	}
 	c.cachedCluster = cluster
-	c.coordinator = newCoordinator(c.cachedCluster, c.s.scheduleOpt, c.s.hbStreams, namespace.DefaultClassifier)
+	c.coordinator = newCoordinator(c.cachedCluster, c.s.scheduleOpt, c.s.hbStreams, newTableNamespaceClassifier(c.cachedCluster.namespacesInfo, core.DefaultTableIDDecoder))
```

Comment:
How about add a config to choose namespace classifier?

Reply:
Agree, I'll add an option in configuration.